core/src/Kokkos_HPX.hpp

```diff
@@ -194,7 +194,7 @@ class HPX {
                                   const bool /* verbose */ = false) {
     std::cout << "HPX backend" << std::endl;
   }
-  uint32_t impl_instance_id() const noexcept { return 0 }
```

Comment:
That never compiled, did it?

Reply:
nope. Only one machine internally tests HPX and because of a general breakdown in internally testing based on job scheduling pause due to full file systems we didn't see it ..